Gordon360/Models/CCT_DB_MODELS.Context.cs

```diff
@@ -59,28 +69,26 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
         public virtual DbSet<PART_DEF> PART_DEF { get; set; }
         public virtual DbSet<RoomAssign> RoomAssign { get; set; }
         public virtual DbSet<Student> Student { get; set; }
-        public virtual DbSet<StudentNews> StudentNews { get; set; }
```

Comment:
Similar to above, these StudentNews tables don't exist in production. As you said, we pull StudentNews from MyGordon.